### ISSUE TITLE
[FEATURE REQUEST] Integration with Paperless NGX

### ISSUE THREAD

**Issue by thomas-mc-work** [Akylas/OSS-DocumentScanner#261]
Hello developer team!

Thank you very much for this project!

**Is your feature request related to a problem? Please describe.**

Not related to a problem, but a dream: I'm using [Paperless NGX](https://github.com/paperless-ngx/paperless-ngx/), which is a wonderful backend to store scanned documents. So far I've been using [Paperless mobile](https://github.com/astubenbord/paperless-mobile/), which is a nice Android client, but sadly lacks a bit behind in active development (not sure, with I think I saw the developer telling this in as issue, too). Then there is OSS Document Scanner. From my perspective it could be a good replacement to Paperless mobile, if it would support the backend as a default storage location. I've seen [you've been considering this yet](https://github.com/Akylas/OSS-DocumentScanner/issues/74#issuecomment-1883629305). You were stating that the API is missing an option to update existing documents.

**Describe the solution you'd like**

I wish the integration could be taken into account again. Maybe without that special API endpoint in the first milestone. A simple upload as a default storage location would be perfectly fine for me (and I guess for many other users too). Together, your project could build up a beautiful ecosystem for free and libre document scanning!

**Describe alternatives you've considered**

I saw that you've already implemented a sharing target for Paperless mobile.

**Additional context**

The client implementation for the backend API can hopefully be taken from the Paperless mobile app as a starting point. So you don't need to start from scratch. It'll nevertheless probably need some polish as time passes.

**Comment by farfromrefug** [comment]
@thomas-mc-work Yes the backend API is lacking in some places and the devs were not really willing to "open" it. I might to think of another way to do it.
I am working on syncing data/image/pdf for webdav/folder. So you could use webdav to sync to a remote folder where paperless would pick them up.
Really not done yet as i have some issues to figure out( specifically how to know a pdf/image needs to be updated).

**Comment by pagdot** [comment]
Would implementing this as "Export to Paperless"be an option? So not syncing, just a share/export to paperless?
